CHANGELOG.md

```diff
@@ -189,8 +255,17 @@
 * Use hidapi's hidraw backend instead of libusb on Linux [#4054](https://github.com/mixxxdj/mixxx/pull/4054)
 * Consistently use "mapping" instead of "preset" to refer to controller mappings [#3472](https://github.com/mixxxdj/mixxx/pull/3472)
 * Introduce new control objects `[Master],indicator_250millis` and `[Master],indicator_500millis` [#4157](https://github.com/mixxxdj/mixxx/pull/4157)
+* Introduce new control object `[Library],clear_search` [#4331](https://github.com/mixxxdj/mixxx/pull/4331)
+* Introduce new control object `[Library],focused_widget` to focus library directly [#4369](https://github.com/mixxxdj/mixxx/pull/4369) [#4490](https://github.com/mixxxdj/mixxx/pull/4490)
 * Don't automatically enable controller if it was disabled before [#4244](https://github.com/mixxxdj/mixxx/pull/4244) [lp:1941042](https://bugs.launchpad.net/mixxx/+bug/1941042)
-* Roland DJ-505: Use new ControlIndicator COs for blinking lights [#4159](https://github.com/mixxxdj/mixxx/pull/4159)
+* Enable Qt logging categories for controller logging (--controllerDebug remains). [#4523](https://github.com/mixxxdj/mixxx/pull/4523)
+* Fix segfault during Mixxx shutdown due to a stale controller connection [#4476](https://github.com/mixxxdj/mixxx/pull/4476) [lp:1946581](https://bugs.launchpad.net/mixxx/+bug/1946581)
+* Components JS: Fix syncbutton [#4329](https://github.com/mixxxdj/mixxx/pull/4329)
```

Comment:
What was fixed? How was is broken?

Reply:
Components JS: Fix sync button longpress function